### PR TITLE
lb_4

### LINKED ISSUE #250: lb_4
Description: Creating a color bar with the labels inside each box. Adding a title to the right of the color bar

NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/lb_4.ncl
NCL Plot: https://www.ncl.ucar.edu/Applications/Images/lb_4_lg.png

Category: Contour

### PR DESCRIPTION
**Pull request by hCraker**
Closes #250 
Original NCL plot:
image

Python recreation:
image

### CODE REVIEW

File: Plots/Contours/NCL_lb_4.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+# default is pointed ends to show there are values beyond the given contour
+# levels
+cbar = plt.colorbar(contour,
+                    ax=ax,
+                    orientation='horizontal',
+                    shrink=0.5,
+                    pad=0.11,
+                    extendrect=True,
+                    extendfrac='auto',
+                    aspect=11)
```
Comment (anissa111) [inline Plots/Contours/NCL_lb_4.py:110]: You could add drawedges=True to recreate the color separations on the NCL colorbar

Comment (hCraker) [reply Plots/Contours/NCL_lb_4.py]: I just changed it. Thanks for pointing it out

**Comment by pilotchute** [comment]
@hCraker make one more push (can even be empty), and you'll be good to go.

**Comment by pilotchute** [comment]
please make an empty commit to trigger the docs build now that the master fix has been merged. :)

**Comment by hCraker** [comment]
@pilotchute I just made a small change but the checks aren't running. Did I do what you wanted me to or did I make a mistake?

**Pull request merged**